source/PluginDev/Assets/GooglePlayGames/Editor/GPGSAndroidSetupUI.cs

```diff
@@ -104,7 +130,9 @@ public static bool PerformSetup(string appId, string nearbySvcId)
             }
 
             GPGSProjectSettings.Instance.Set("proj.AppId", appId);
+			GPGSProjectSettings.Instance.Set(GPGSUtil.ANDROIDCLIENTIDKEY, clientId);
```

Comment:
If it is easy to fix the indentation, please do, but if it is a pain, don't worry about it.